docs/rules/object-curly-spacing.md

```diff
@@ -22,28 +22,28 @@ export { foo };
 
 ## Rule Details
 
-This rule aims to maintain consistency around the spacing inside of object literals. It also
-applies to EcmaScript 6 destructured assignment and import/export specifiers.
-
-It either requires or disallows spaces between those braces and the values inside of them.
-Braces that are separated from the adjacent value by a new line are exempt from this rule.
+This rule enforce consistent spacing inside braces of object literals, destructuring assignments, and import/export specifiers.
 
 ## Options
 
-There are two main options for the rule:
+This rule has a string option:
```

Comment:
What if we added language like so:

```
This rule has a string option:

*Stuff*

And an object option:

*More stuff*
```

This would make the language consistent with other forms that we currently use, such as:

```
This rule has a string option:

*Stuff*

Or an object option:

*More stuff*
```